contracts/TokenConversionLibTest.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity 0.4.23;
+
+import "./TokenConversionLib.sol";
+
+library TokenConversionLibTest {
```

Comment:
This file does not include any tests, why is it called `...Test`? What's our convention here? @pgev?